src/CommunityToolkit.Maui.Core/Essentials/FileFolderDialog.tizen.cs

```diff
@@ -0,0 +1,398 @@
+		var radius = 8d.ToPixel();
+
+		var content = new View
+		{
+			CornerRadius = radius,
+			BoxShadow = new Shadow(20d.ToPixel(), TColor.Black.ToNative()),
+			Layout = new LinearLayout
+			{
+				VerticalAlignment = VerticalAlignment.Center,
+				HorizontalAlignment = HorizontalAlignment.Center,
+				LinearOrientation = LinearLayout.Orientation.Vertical,
+			},
+			SizeWidth = Window.Instance.WindowSize.Width * (isHorizontal ? 0.5f : 0.8f),
+			BackgroundColor = TColor.White.ToNative(),
+		};
+
+		content.Add(new Label
+		{
+			Text = isFileSelectionMode ? fileSaveString : selectFolderString,
+			Margin = new Extents(margin1, margin1, margin1, margin2),
+			WidthSpecification = LayoutParamPolicies.MatchParent,
+			HorizontalTextAlignment = Tizen.UIExtensions.Common.TextAlignment.Start,
+			VerticalTextAlignment = Tizen.UIExtensions.Common.TextAlignment.Center,
+			FontAttributes = Tizen.UIExtensions.Common.FontAttributes.Bold,
+			TextColor = TColor.Black,
+			PixelSize = 21d.ToPixel(),
+		});
+
+		content.Add(new View
+		{
+			BackgroundColor = TColor.FromHex("#cccccc").ToNative(),
+			SizeHeight = 1.5d.ToPixel(),
+			WidthSpecification = LayoutParamPolicies.MatchParent,
+		});
+
+		directoryScrollView = new ScrollView
+		{
+			Margin = new Extents(margin1, margin1, 0, 0),
+			VerticalScrollBarVisibility = Tizen.UIExtensions.Common.ScrollBarVisibility.Default,
+			WidthSpecification = LayoutParamPolicies.MatchParent,
+		};
+		directoryScrollView.ContentContainer.Layout = new LinearLayout
+		{
+			LinearOrientation = LinearLayout.Orientation.Vertical,
+		};
+		content.Add(directoryScrollView);
+
+		UpdateDirectoryScrollView(initialFileFolderPath);
+
+		content.Add(new View
+		{
+			BackgroundColor = TColor.FromHex("#cccccc").ToNative(),
+			SizeHeight = 1.5d.ToPixel(),
+			WidthSpecification = LayoutParamPolicies.MatchParent,
+		});
+
+		if (isFileSelectionMode)
+		{
+			var newFolderButton = new Button
+			{
+				Focusable = true,
+				Text = "New Folder",
+				TextColor = TColor.Black,
+				BackgroundColor = TColor.Transparent.ToNative(),
+				Margin = new Extents(margin1, margin1, 0, 0),
+				WidthSpecification = LayoutParamPolicies.MatchParent
+			};
+			newFolderButton.Clicked += async (sender, args) =>
+			{
+				try
+				{
+					var newDirectoryName = await new PromptPopup("New Folder Name", "").Open();
+					var newDirectoryPath = IsDirectory(selectedPath) ? selectedPath : Path.GetDirectoryName(selectedPath);
+					if (string.IsNullOrEmpty(newDirectoryPath) || string.IsNullOrEmpty(newDirectoryName))
+					{
+						return;
+					}
+					newDirectoryPath = newDirectoryPath.EndsWith(slashSymbol) ? newDirectoryPath : newDirectoryPath + slashSymbol;
+					if (CreateSubDirectory(newDirectoryPath + newDirectoryName))
+					{
+						UpdateDirectoryScrollView(selectedPath);
+					}
+				}
+				catch (Exception ex)
+				{
+					new Tizen.Applications.ToastMessage
+					{
+						Message = $"Failed to create new folder, {ex.Message}"
```

Comment:
This isn't localized

Reply:
Tizen doesn't have the way to get the localized text from the system that's commonly available on all target device profiles... I think I will think about how I can handle this. Or maybe providing the methods or properties to get the text.

Reply:
I added it for 2 strings: Save and OK.
I haven't found a better solution rather then creating a static method to setup values